src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Snapshot/TargetedDependenciesSnapshot.cs

```diff
@@ -32,6 +32,20 @@ internal class TargetedDependenciesSnapshot : ITargetedDependenciesSnapshot
             }
         }
 
+        /// <summary>
+        /// Internal for testing.
+        /// </summary>
+        internal TargetedDependenciesSnapshot(IDictionary<string, IDependency> dependenciesWorld, IEnumerable<IDependency> topLevelDependencies)
```

Comment:
I would rather make this constructor private, with in `internal static` factory method with a clear name like `CreateForUnitTestingOnly` or something.